src/GraphODataTemplateWriter/CodeHelpers/PHP/TypeHelperPHP.cs

```diff
@@ -68,6 +68,7 @@ public static string GetTypeString(this OdcmType @type)
                 case "Int64":
                     return "int";
                 case "Double":
+                case "Single":
```

Comment:
FYI per https://www.php.net/manual/en/function.gettype.php.

> "double" (for historical reasons "double" is returned in case of a float, and not simply "float")

I don't think this changes anything. No action needed. It is all the same under the covers.